src/ResourceManager/FrontDoor/Commands.FrontDoor/Az.FrontDoor.psd1

```diff
@@ -15,7 +15,7 @@
 ModuleVersion = '0.1.0'
```

Comment:
I'd recommend keeping this the same as all other Az module versions, `0.5.0`. If you're looking to release it for the first time, maybe make it `0.5.1` since it missed the `0.5.0` module set?
We'll have to decide a version strategy for preview modules prior to Az 1.0 release. For now, let's keep them in-line with the rest of the Az modules (what we've been doing up until now).

Reply:
We need to be super sure that we only release stable modules as "1.0.0" in December - it would be bad if all of these preview modules that are supposed to stay preview get released as stable. Fixed for the time being though.